packages/node/src/sdk/index.ts

```diff
@@ -113,12 +113,12 @@ export function initWithoutDefaultIntegrations(options: NodeOptions | undefined
  * Initialize Sentry for Node, without performance instrumentation.
  */
 function _init(
-  options: NodeOptions | undefined = {},
+  _options: NodeOptions | undefined = {},
   getDefaultIntegrationsImpl: (options: Options) => Integration[],
 ): NodeClient {
-  const clientOptions = getClientOptions(options, getDefaultIntegrationsImpl);
+  const options = getClientOptions(_options, getDefaultIntegrationsImpl);
```

Comment:
You don't really see this here, but we've been checking properties further below in `init` based off `options`, not `clientOptions`, leading to us not using the defaults defined in `getClientOptions`. IMHO it is confusing to have to options objects and check everything on `clientOptions` (which is also why this even happened in the first place...) so I changed this to just call this `options` and use it everywhere (we never really need to access the unprocessed options again afterwards anyhow...)